src/main/java/MainApp.java

```diff
@@ -19,13 +29,64 @@ public void setUp() {
         options.addArguments("disable-infobars");
         driver = new ChromeDriver(options);
         wait = new WebDriverWait(driver, 60);
+
+        signInPage = new SignInPage(driver);
+        header = new Header(driver);
+        indexPage = new IndexPage(driver);
+        signInPage = new SignInPage(driver);
+        header = new Header(driver);
```

Comment:
is it layer card or layer cart (t or d)?

Reply:
done